javaslang/src-gen/main/java/javaslang/Tuple2.java

```diff
@@ -133,15 +135,25 @@ public T2 _2() {
         return new Tuple2<>(_1, value);
     }
 
+     /**
+      * Swaps the elements of this {@code Tuple}.
+      *
+      * @return A new Tuple where the first element is the second element of this Tuple
+      *   and the second element is the first element of this Tuple.
+      */
+     public Tuple2<T2, T1> swap() {
+       return Tuple.of(_2, _1);
+     }
+
     /**
-     * Swaps the elements of this {@code Tuple}.
-     *
-     * @return A new Tuple where the first element is the second element of this Tuple
-     *   and the second element is the first element of this Tuple.
-     */
-    public Tuple2<T2, T1> swap() {
-      return Tuple.of(_2, _1);
-    }
+    * Converts the tuple to java.util.Map.Entry {@code Tuple}.
+    *
+    * @return A  java.util.Map.Entry where the first element is the key and the second
+    * element is the value.
+    */
+     public Map.Entry<T1, T2> toEntry(){
+         return new AbstractMap.SimpleEntry<>(_1, _2);
+     }
```

Comment:
The changes look good to me, thank you!
We are nearly there.
Before I merge, please align the lines right. The indentation needs to be fixed in the generator.
See also https://github.com/ashrko619/javaslang/blob/26257170f75eac814354e5f27bca6c48a7455a52/javaslang/src-gen/main/java/javaslang/Tuple2.java#L138-L156